explorer/src/Pos/Explorer/ExtraContext.hs

```diff
@@ -172,3 +182,14 @@ instance (Monad m, MonadDBRead m, MonadSlotsData ctx m) =>
         extraCtx <- Ether.ask @ExtraContext
         let explorerMockMode = ecExplorerMockableMode extraCtx
         emmGetLeadersFromEpoch explorerMockMode $ epochIndex
+
+    getEpochBlocksCSLI epoch page = do
+        extraCtx <- Ether.ask @ExtraContext
+        let explorerMockMode = ecExplorerMockableMode extraCtx
+        (emmGetEpochBlocks explorerMockMode) epoch page
+
+    getEpochPagesCSLI epoch = do
+        extraCtx <- Ether.ask @ExtraContext
+        let explorerMockMode = ecExplorerMockableMode extraCtx
+        emmGetEpochPages explorerMockMode $ epoch
```

Comment:
> emmGetEpochPages explorerMockMode $ epoch

why `$` here ?

Reply:
The same applies for others in the same module

Reply:
Also, there is a lot of copy paste in this module. I see 
```
+        extraCtx <- Ether.ask @ExtraContext
+        let explorerMockMode = ecExplorerMockableMode extraCtx
```
being used at 7 places

Reply:
That's two lines on 7 places - hardly a lot. 
I saw that, and understand why you would comment that, but I decided that it's more understandable this way, I really don't think we should replace 7-14 lines of code with something that's less understandable.
I really think there is a fine line between removing lines of code and making something easier to read.

Reply:
```haskell
someFunctionName someVariable = ecExplorerMockableMode <$> Ether.ask @ExtraContext
```
how is this confusing?

>  it's more understandable this way, I really don't think we should replace 7-14 lines of code with something that's less understandable. I really think there is a fine line between removing lines of code and making something easier to read.

and more error prone and harder to maintain

Reply:
@akegalj Why don't you try writing that function? And compiling? If you succeed, you can push the changes, but it's hard for me to explain something if you didn't try it yourself.